BUILD

```diff
@@ -94,11 +94,15 @@ kernel(
 		"amd64/bin/date": "bin/date",
```

Comment:
where are these things being moved? Maybe I am just not understanding what you're doing here. I don't get it.